entities.py

```diff
@@ -64,6 +65,54 @@ def distance_to(self, dest):
         return (self.pos - dest).norm
 
 
+class Item(Entity):
+
+    def __init__(self, name, pos, char, color, blocks, effect=None):
+        super().__init__(name, pos, 'item', char, color, blocks, RenderPriority.ITEM)
```

Comment:
Maybe default blocks to False and not have a blocks parameter in Item?

Reply:
Also, did you create RenderPriority.ITEM?

Reply:
well I thought of potentially having items such as chests in the world, which are `blocks=True` and you created RenderPriority.ITEM 🤣